systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -452,9 +452,9 @@ def fw_vpcrouter(self):
             self.fw.append(
                 ["filter", "", "-A INPUT -i %s -p udp -m udp --dport 67 -j ACCEPT" % self.dev])
             self.fw.append(
-                ["filter", "", "-A INPUT -i %s -p udp -m udp --dport 53 -j ACCEPT" % self.dev])
+                ["filter", "", "-A INPUT -i %s -p udp -m udp --dport 53 -s %s -j ACCEPT" % (self.dev, self.address['network'])])
             self.fw.append(
-                ["filter", "", "-A INPUT -i %s -p tcp -m tcp --dport 53 -j ACCEPT" % self.dev])
+                ["filter", "", "-A INPUT -i %s -p tcp -m tcp --dport 53 -s %s -j ACCEPT" % (self.dev, self.address['network'])])
```

Comment:
@rhtyd I agree that keeping it limited to the scope of the PR is the most pragmatic approach.  The intention of my comment was to encourage test case improvement  in small increments.